scripts/changelog.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
```

Comment:
@levithomason Here's an initial version, if you'd like to take a look. I believe you and @ariporad should automatically be collaborators, so feel free to make changes.

https://github.com/shelljs/release/blob/master/index.js

I also have a similar one for changelog.js if you'd like to base your module off of that. It still relies on `curl`, however.